src/Objects/Image.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of the fusonic/webapp package.
+ *
+ * (c) Fusonic GmbH <office@fusonic.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Fusonic\WebApp\Objects;
+
+use Fusonic\WebApp\Members\PlatformInterface;
+use Fusonic\WebApp\Members\PlatformTrait;
+
+/**
+ * Represents an <b>image object</b>.
+ *
+ * @package Fusonic\WebApp
+ *
+ * @see https://www.w3.org/TR/appmanifest/#image-object-and-its-members
+ */
+final class Image implements PlatformInterface
```

Comment:
How about `PlatformMemberInterface`? The spec almost always uses the term "platform member".